Examples/UIExplorer/StatusBarExample.js

```diff
@@ -293,6 +305,21 @@ exports.examples = [{
   },
 }];
 
+if (android) {
```

Comment:
The issue is that StatusBar has a lot of platform specific props and I wanted to only show the ones that are relevant to the current platform in the example but I agree it does make the code more complicated than it should be. Maybe having one example component per prop is the best as every example will be smaller.